planning/scenario_planning/parking/costmap_generator/include/costmap_generator/costmap_generator.hpp

```diff
@@ -42,11 +42,12 @@
  *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
I found it a bit strange that there are two copyright headers, but they seem to be there in master already. Is that correct?

Reply:
I think the very original code was copied from Autoware.AI, which was first made by Nagoya University, and was modified for this project. That's why we had to leave the old license for copied codes.